core/src/main/java/org/elasticsearch/common/geo/builders/ShapeBuilder.java

```diff
@@ -77,16 +77,10 @@
     /** @see com.spatial4j.core.shape.jts.JtsGeometry#index() */
     protected final boolean autoIndexJtsGeometry = true;//may want to turn off once SpatialStrategy impls do it.
 
-    protected Orientation orientation = Orientation.RIGHT;
-
     protected ShapeBuilder() {
```

Comment:
Hmm... I wonder if we should add:
```
assumeTrue("JTS not available", ShapesAvailability.JTS_AVAILABLE);
```
to ```ShapeBuilder``` since JTS and Spatial4J are optional dependencies. I haven't tested everything after removing JTS and Spatial4J from the classpath. I know this was previously a problem for plugins, but its only caught in ```GeoShapeQueryBuilder```